### DISCUSSION TITLE
？

### DISCUSSION THREAD

**Discussion by AntEffect** [WeNeedHome/SummaryOfLoanSuspension#17] [General]
编辑了，小粉红太可怕

**Comment by dyd110** [comment]
就算是小粉红，事情砸到自己头上的时候也会跳出来的，毕竟喊口号容易，真正在这个地方活着，很难

**Reply by cutemaoqiu** [reply]
砸到头上了世界上就少一个小粉红了

**Comment by hadesII** [comment]
烂尾楼里小红粉怕是难有

**Comment by Qianzf** [comment]
提这个问题的人除了会挑拨，激化矛盾。可还能学着干点人做的事吗？人家买了烂尾房的已经够惨的了，他们需要的是解决问题的方法！不是来给你做情绪垃圾桶的。
感觉你们反共反的已经失了智。连作为人的基本的同理心都已经没了。

**Reply by PCanyi** [reply]
解决问题的方法是你出钱把烂尾房盖好。

**Reply by Qianzf** [reply]
> 解决问题的方法是你出钱把烂尾房盖好。

阴阳怪气并不能体现你做人的素养。哪怕你能提点对政府施压的方法我也会给你点个赞，这种垃圾言论建议你还是自己留着就行

**Reply by Qianzf** [reply]
> > 提这个问题的人除了会挑拨，激化矛盾。可还能学着干点人做的事吗？人家买了烂尾房的已经够惨的了，他们需要的是解决问题的方法！不是来给你做情绪垃圾桶的。
> > 感觉你们反共反的已经失了智。连作为人的基本的同理心都已经没了。
> 
> 道德高地凉快吗？

我不知道我这个算不算讲道德，但是我做事讲个问心无愧，一码事就作为一码事处理。
不过我真心觉得，如果符合传统道德的评论都要被人制裁说是站在道德高地而不能出声的社会，没有存在的必要。
符合道德，我哪怕就站在道德高地，那我不仅要说，更要大声的说。

**Reply by Qianzf** [reply]
> > > > 提这个问题的人除了会挑拨，激化矛盾。可还能学着干点人做的事吗？人家买了烂尾房的已经够惨的了，他们需要的是解决问题的方法！不是来给你做情绪垃圾桶的。
> > > > 感觉你们反共反的已经失了智。连作为人的基本的同理心都已经没了。
> > > 
> > > 
> > > 道德高地凉快吗？
> > 
> > 
> > 我不知道我这个算不算讲道德，但是我做事讲个问心无愧，一码事就作为一码事处理。
> > 不过我真心觉得，如果符合传统道德的评论都要被人制裁说是站在道德高地而不能出声的社会，没有存在的必要。
> > 符合道德，我哪怕就站在道德高地，那我不仅要说，更要大声的说。
> 
> 哈哈哈急了，你就是小粉红本红吧？

你要认为我是，那我便是就是了。

**Reply by danuonuo** [reply]
道德制高点不错哈

**Reply by mulubanche** [reply]
搞对立是这批混子最大的动力，烂尾是小粉红搞的吗？是小粉红拥立的吗？是小粉红支持的吗？如果这个屎盆子往小粉红身上推，这人不是蠢就是坏。

**Reply by hadesII** [reply]
如果不是粉红整天助纣为虐，分化群众，他们敢这么丧心病狂吗

**Reply by Ezail3** [reply]
> > 解决问题的方法是你出钱把烂尾房盖好。
> 
> 阴阳怪气并不能体现你做人的素养。哪怕你能提点对政府施压的方法我也会给你点个赞，这种垃圾言论建议你还是自己留着就行

你说的政府是哪个政府？美国政府反正是从来不听老百姓建议的，都是一手遮天

**Reply by luuthink** [reply]
> 提这个问题的人除了会挑拨，激化矛盾。可还能学着干点人做的事吗？人家买了烂尾房的已经够惨的了，他们需要的是解决问题的方法！不是来给你做情绪垃圾桶的。 感觉你们反共反的已经失了智。连作为人的基本的同理心都已经没了。

最大的问题是什么不用我多说了吧？美国政府我劝你不要搬起石头砸自己的脚，把石头给俺们老乡盖房子多中嘞

**Reply by luuthink** [reply]
> > 解决问题的方法是你出钱把烂尾房盖好。
> 
> 阴阳怪气并不能体现你做人的素养。哪怕你能提点对政府施压的方法我也会给你点个赞，这种垃圾言论建议你还是自己留着就行

跟政府施压，用什么？你说到法治，我就觉得好笑

**Reply by spartawhy117** [reply]
> > 提这个问题的人除了会挑拨，激化矛盾。可还能学着干点人做的事吗？人家买了烂尾房的已经够惨的了，他们需要的是解决问题的方法！不是来给你做情绪垃圾桶的。
> > 感觉你们反共反的已经失了智。连作为人的基本的同理心都已经没了。
> 
> 道德高地凉快吗？

遇到这种事情第一个想到的居然是小粉红挨锤了也是牛逼，那每天全中国发生的操蛋事情能让你赢10年了。说好的关爱具体的人呢.jpg

**Reply by CodeJiang** [reply]
这个时候还说风凉话，搞内部斗争的不是蠢就是坏。

**Reply by spartawhy117** [reply]
> > 这个时候还说风凉话，搞内部斗争的不是蠢就是坏。
> 
> 小粉红能算自己人？

你这三观理解的社会运行逻辑是只有粉红和非粉红在运行 社会是吧，某些既得利益群体都跟你阶级隔离了，还给他们也套个小粉红的身份标签，人家鸟你吗？

**Comment by XiaoyaoEzio** [comment]
购房者里哪怕有，这一拳下来估计也不红了，至于看乐子的，小粉红看不到这些

**Reply by zhaokai5520** [reply]
小红粉都是分配房子的吗

**Comment by chen-wbc** [comment]
有10年老粉红，你怕不怕啊就问

**Comment by Krocsky** [comment]
其实不要给人贴标签🏷，都是国人，小粉红那种都是不经事的或者碰不到这些事的。如果真的碰到了，也一样会恶心的

**Reply by Krocsky** [reply]
我希望能够在这里看到，大家集思广益解决问题的，而不是在这里搞对立搞站队。咱帮不上忙的，尽量别添乱

**Reply by chen-wbc** [reply]
关羽买到烂尾楼这种事情怎么冷静

**Reply by GengJian** [reply]
> 

借酒消愁，越想越不冷静，就斩了华雄

**Reply by Krocsky** [reply]
> 关羽买到烂尾楼这种事情怎么冷静

我真的，我的房子也快交付了，要是烂尾了，银行、开发商我都不让他们睡安稳觉，他们也是有家人的（我这种就比较极端了，但是目前感觉也想不到啥好办法）

**Comment by 457694569** [comment]


**Reply by chen-wbc** [reply]
来来来 主席给你当

**Comment by bayshier** [comment]


**Reply by wsgouwan** [reply]
10年前 黑伟人的不要太多了  怎么现在想起他来了？

**Reply by Swiftfrog** [reply]
你看毛粉来了。。。何况小粉红。

**Reply by zhaokai5520** [reply]
从上周末郑州维权图片里显示，怀念他老人家的人还挺多

**Comment by bladechen** [comment]
大家好， 我是美分

**Comment by xyouZone** [comment]
这个评论区是讨论问题的，不是给你嘲笑和发泄情绪的

**Comment by wlcp** [comment]
你是不是以为你很幽默？？、！

**Comment by zhenghao11** [comment]
小粉红都急的找美领馆求助了,哪还有心情来这里。

**Comment by 378804441** [comment]
啧啧啧，本来还挺同情你的。在这搞这种阴阳怪气的，不针对别人 只针对你个人，你多少有点活该了。
这么跟你讲，当小粉红总比当 精神美国人 精神日本人强。你也配姓赵？

**Reply by AntEffect** [reply]
哎呀，小粉红出现了

**Reply by luuthink** [reply]
> 啧啧啧，本来还挺同情你的。在这搞这种阴阳怪气的，不针对别人 只针对你个人，你多少有点活该了。 这么跟你讲，当小粉红总比当 精神美国人 精神日本人强。你也配姓赵？

你也配姓赵不是就是说小粉红的吗？搬起石头砸自己的脚是吧

**Reply by 378804441** [reply]
所以说你没文化呢。。。你也配姓赵的出处都不知道。
成为精美精日之前先好好学学9年教育吧😂 
你也配姓赵？

**Reply by luuthink** [reply]
> 所以说你没文化呢。。。你也配姓赵的出处都不知道。 成为精美精日之前先好好学学9年教育吧😂 你也配姓赵？

笑死，看看谁才是赵家人咯 https://github.com/programthink/zhao ，你也配姓赵？

**Comment by Myzhengzheng** [comment]
> 烂尾楼里会出现小粉红吗？

垃圾到哪里都讨人厌，你就是这种。

**Comment by jokerdrake** [comment]
话说，这波，江西的咖位这么高吗？直接放第一个。

**Reply by realwugang** [reply]
革命根据地的老表可不是闹着玩的

**Reply by longalei** [reply]
你想影射啥？要不我从小到大排个序？
